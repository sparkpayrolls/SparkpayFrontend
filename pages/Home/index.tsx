import { Button } from '@/components/Button/Button';
import type { NextPage } from 'next';
import Head from 'next/head';
import Image from 'next/image';
import Link from 'next/link';
import dashboard_preview from '../../public/svgs/group-33971.svg';
import dashboard_preview2 from '../../public/svgs/frame-11825.svg';

const Home: NextPage = () => {
  return (
    <div>
      <Head>
        <title>SparkPay</title>
        <meta name="description" content="Sparkpay homepage" />
        <link rel="icon" href="/favicon.ico" />
      </Head>

      <main className="home">
        <section className="hero-section">
          <caption className="hero-section__title">
            Payroll Made <span className="hero-section__title--em">Easy</span>
          </caption>
          <p className="hero-section__subtext">
            Lorem ipsum dolor amet, consectetur adipiscing accumsan pellentesque
            volutpat.
          </p>

          <div className="hero-section__join-list">
            <input
              type="email"
              name="email"
              className="hero-section__join-list--input"
              placeholder="Enter your email address"
            />

            <Button
              label="Join Waitlist"
              type="button"
              onClick={() => {}}
              primary={true}
            />
          </div>
        </section>

        <section className="app-preview">
          <Image
            src={dashboard_preview}
            alt="app preview"
            className="app-preview__image"
          />
        </section>

        <section className="features">
          <div>
            <h2 className="features__title">
              Focus on your business while we handle your Payroll.
            </h2>
            <p className="features__subtext">
              Adipiscing tortor, pellentesque donec deaut accumsan nibh turpis
              pellentesque donec deaut consectetur.
            </p>

            <ul className="features__list">
              <li className="features__list-item">
                Lorem ipsum sit amet adipiscing.
              </li>
              <li className="features__list-item">Adipiscing tortor.</li>
              <li className="features__list-item">
                Pellentesque donec accumsan.
              </li>
              <li className="features__list-item">
                Pellentesque donec accumsan.
              </li>
            </ul>

            <Link href="/create-account">
              <a className="features__link">Get Started</a>
            </Link>
          </div>

          <div className="features__image">
            <Image src={dashboard_preview2} alt="dashboard preview" />
          </div>
        </section>
      </main>
    </div>
  );
};

export default Home;
