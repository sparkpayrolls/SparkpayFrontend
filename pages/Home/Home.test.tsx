import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

it('renders homepage correctly', () => {
  const app = renderer.create(<Home />).toJSON();
  expect(app).toMatchSnapshot();
});
