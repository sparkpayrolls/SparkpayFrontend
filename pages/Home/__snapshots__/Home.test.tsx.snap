// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly 1`] = `
<div>
  <h1>
    This is the home page
  </h1>
</div>
`;
